bindata/manifests/daemon/daemonset.yaml

```diff
@@ -92,6 +92,23 @@ spec:
               mountPath: /host/etc/os-release
               readOnly: true
         {{- end }}
+        {{- if ne .RDMACNIImage "" }}
+        - name: rdma-cni
+          image: {{.RDMACNIImage}}
+          command:
+            - /bin/sh
```

Comment:
please let's not have a copy like this here.

can we create an entrypoint.sh and use it in the image for the rdma-cni repo (I can review and approve it)

the same way we do for the sriov-cni